js/tests/unit/util/scrollbar.spec.js

```diff
@@ -4,10 +4,13 @@ import Manipulator from '../../../src/dom/manipulator'
 
 describe('ScrollBar', () => {
   let fixtureEl
+  const doc = document.documentElement
   const parseInt = arg => Number.parseInt(arg, 10)
-  const getRightPadding = el => parseInt(window.getComputedStyle(el).paddingRight)
+  const getPaddingX = el => parseInt(window.getComputedStyle(el).paddingRight)
```

Comment:
IMO the old name was more descriptive. What means X here? Horizontal?

Reply:
Yes, is horizontal. trying to see forward, scrollbar doesn't support yet modifications on RTL that are done on left side of the screen. So I decided to rename the function to something more generic for later usage. I am ok to revert it if you find it better

Reply:
I already suspected that this might be the reason, but I'd rather call it `getHorizonalPadding` or `getXPadding` if you feel that's too long. Can't say why, but `getPaddingX` feels wrong to me. (On a sidenote, `window.pageXOffset` follows the same scheme.)